akka-javasdk-parent/pom.xml

```diff
@@ -82,6 +82,8 @@
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
         <maven-deploy-plugin.version>3.1.1</maven-deploy-plugin.version>
 
+        <!-- FIXME must be aligned with sdk/runtime/akka-grpc -->
+        <protobuf-java.version>3.25.5</protobuf-java.version>
```

Comment:
Is this a FIXME or a reminder for the future? Can we add a reminder in the sbt build too? Is it when bumping akka-grpc plugin that we should remember to bump here too?

Reply:
This would optimally be automatic, like we do with a few other things. I'll check if that is possible somehow.

Reply:
sorted in [b52fce2](https://github.com/akka/akka-sdk/pull/1032/commits/b52fce27923d0fd7b14af6dbe11da0172725881a)